aspnetcore/signalr/dotnet-client.md

```diff
@@ -34,7 +32,24 @@ Install-Package Microsoft.AspNetCore.SignalR.Client
 
 To establish a connection, create a `HubConnectionBuilder` and call `Build`. The hub URL, protocol, transport type, log level, headers, and other options can be configured while building a connection. Configure any required options by inserting any of the `HubConnectionBuilder` methods into `Build`. Start the connection with `StartAsync`.
 
-[!code-csharp[Build hub connection](dotnet-client/sample/signalrchatclient/MainWindow.xaml.cs?name=snippet_MainWindowClass&highlight=14-16,32)]
+[!code-csharp[Build hub connection](dotnet-client/sample/signalrchatclient/MainWindow.xaml.cs?name=snippet_MainWindowClass&highlight=15-17,42)]
+
+### Handle lost connection
+
+You can use the `Closed` event to respond to a lost connection. For example, you might want to automate reconnection.
+
+The `Closed` event requires a delegate that returns a `Task`. The reason for that return type is to allow you to run async code without having to use `async void`. If your `Closed` handler doesn't use `await`, you can return `Task.CompletedTask` to satisfy the delegate signature, as in the following example:
```

Comment:
Use the words "you" and "your" sparingly. Here's a suggested rework on the last 2 sentences:

~~The reason for that return type is to allow you to run async code without having to use `async void`. If your `Closed` handler doesn't use `await`, you can return `Task.CompletedTask` to satisfy the delegate signature, as in the following example:~~
The `Task` return type allows async code to run without using `async void`. If the `Closed` handler doesn't use `await`, return `Task.CompletedTask` to satisfy the delegate signature. For example: